connectivity/check/check.go

```diff
@@ -64,10 +64,10 @@ type Parameters struct {
 	JunitFile             string
 	JunitProperties       map[string]string
 
-	IncludeUpgradeTest    bool
-	UpgradeTestSetup      bool
-	UpgradeTestResultPath string
-	FlushCT               bool
+	IncludeInterruptionTest    bool
```

Comment:
Nit: I prefixed the related pods and svc with `test-conn-disrupt`. Maybe we can reflect that in the params too? For example, `IncludeConnDisruptTest` (same for other params if agreed).